README.md

```diff
@@ -11,9 +11,11 @@ Make a PR. Fux wit dis repo. Add things. Delete things. Screw things up so you c
 
 ## Rules
 Hereth be yon rules:
-1. Thou shalt ask ONLY ridiculous questions, and ridiculous shall thine questions be.
-2. All Pull Requests must be accompanied by an animated gif. No exceptions.
-3. three sir!
-4. Edit [WeAreGo.md](docs/WeAreGo.md) checking the box next to your name to prove your karate & friendship. (**test if you can commit & push**)
-5. ????
-6. PROFIT!!!
+- Thou shalt ask ONLY ridiculous questions, and ridiculous shall thine questions be.
+- All Pull Requests must be accompanied by an animated gif. No exceptions.
+- three sir!
+- Edit [WeAreGo.md](docs/WeAreGo.md) checking the box next to your name to prove your karate & friendship. (**test if you can commit & push**)
+- All roads lead to Rome
+- Get your hands dirty, try for thirty - After that, ask for Matt
+- ????
+- PROFIT!!!
```

Comment:
Not a blocker, but did you know GH markdown lets you use numbers like this:

1.
1.
1.
1.

Reply:
which is 
```markdown
1.
1.
1.
1.
```